### ISSUE TITLE
Unravel (1225560)

### ISSUE THREAD

**Issue by frostworx** [ValveSoftware/Proton#3961]
Compatibility Report

    Name of the game with compatibility issues: Unravel
    Steam AppID of the game: 1225560

System Information

    GPU: RTX 2080
    Driver/LLVM version: nvidia-vulkan-440.66.15
    Kernel version: 5.6.15-arch1-1
    Proton version: 5.0-8

I confirm:

    [*] that I haven't found an existing compatibility report for this game.
    [*] that I have checked whether there are updates for my system available.

extracted 60mb logfile:

[steam-1225560.log.zip](https://github.com/ValveSoftware/Proton/files/4753387/steam-1225560.log.zip)

Symptoms:

The game fails to start. this has multiple reasons:

* at least proton-5.0-9 required to origin working

* the game fails to start anyway. it opens a black window which quickly is minimized. When you unminimize it it gets fullscreen, you see an orange loading bar, and when that finished it gets stuck with a black screen. you have to kill the process here

**Comment by DanMan** [comment]
I've played this through [Lutris](https://lutris.net/games/unravel/) before directly through Origin so I know it can work. Maybe try some of the [fixes](https://lutris.net/games/install/11440/view)?

**Comment by frostworx** [comment]
good idea, thx!
edit: it was worth a try, but nothing worked.
tried:
  overrides:
    nvapi,nvapi64: disabled
    xaudio2_7: native,builtin
and 
  <Setting key="EnableIgo" value="false" type="1"/>
(damn, I was pretty sure that this could be the problem)
everything else was already applied per default

maybe it has problems with system's gst-plugins-good, at least UnravelTwo has:
https://github.com/ValveSoftware/Proton/issues/3962#issuecomment-641505085
the fix doesn't work here though.

**Comment by DanMan** [comment]
Have you tried installing `vcrun2012`, too?

**Comment by frostworx** [comment]
yes, already tried that as well. It looks like it already gets installed automatically from steam, when I start the installer (protontricks) I only get repair/uninstall/close options. but thanks for trying to help!

**Comment by frostworx** [comment]
Unravel works ootb with proton 5.13-1!
Thank you very much!

**Comment by hcwinsemius** [comment]
Not sure if this topic's still open. But Unravel stopped working. It may have to do with the recent 5.13-3 update. The only thing I see is a small white dot blinking on the bottom right side. Nothing else.

Anyone knows what to do about it?

**Comment by kisak-valve** [comment]
Hello @hcwinsemius, the compatibility report is open as indicated in the top left corner of this issue report. Please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options, reproduce your issue, and attach the generated $HOME/steam-$APPID.log to this issue report as a file. (Proton logs compress well if needed.)

**Comment by hcwinsemius** [comment]
Huh....interesting. Today I got an update to 5.13-4 and....it started working again! Thanks for the response anyway!

**Comment by dariox86** [comment]
From my report on ProtonDB: the game works with Proton-GE 5.21-GE-1. When installing the game for the first time, Origin yields an error message. Click OK and launch the game again. When Origin loading window is shown, you may need to keep moving the mouse and click on random places on the desktop for the game to load. The game will launch minimized. Click the minimized window to begin. If the game is stuck at a loading bar upon launch in full screen and it was previously launched with an older version of Proton, delete the corresponding Proton prefix and start from scratch. The only remaining problem is related to Media Foundation. Video playback works out of the box with Proton-GE 5.21-GE-1, but red color component is missing in cutscenes.

EDIT: with Proton Experimental 5.13-20210107 colors are OK.

**Comment by dariox86** [comment]
For me this is working out of the box with cutscenes with Proton 5.13-5. I think the issue can be closed.

**Comment by RiantAnda** [comment]
I'm having issues with my Xbox controller. 
It seems to be locked in the up and left direction, no proper control of Yarny possible.
Seen with Proton 6.3-2

On startup, there is some kind of test-screen, which indicates issues with video playback.
[steam-1225560.log](https://github.com/ValveSoftware/Proton/files/6391191/steam-1225560.log)
Link to full system information report as [Gist](https://gist.github.com/RiantAnda/baf239d84f8d2984bab911459ec512d6)

**Comment by dariox86** [comment]
@RiantAnda, for the issue with video playback, if you are using Flatpak Proton try using native Proton instead.

**Comment by RiantAnda** [comment]
@dariox86
No, I'm using the steam package from the repo of my distribution (Arch linux). Proton as been installed from within the steam client.

**Comment by dariox86** [comment]
@RiantAnda, try with "Enable Shader Pre-Caching" disabled in the Steam client.

**Comment by RiantAnda** [comment]
@dariox86
This changed the behavior from a test picture to blinking white dots in the lower right corner. Identical to when I tried disabling gstreamer via `WINEDLLOVERRIDES="winegstreamer=d" %command%`.
With this, I'm stuck at this point and can't go into the main menu. 
Strange that "Enable Shader Pre-Caching" interferes with video playback?!?

**Comment by dariox86** [comment]
@RiantAnda, shader pre-caching is known to affect Media Foundation playback in Flatpak Steam. Try to re-enable shader pre-caching and install all the relevant GStreamer multimedia decoders . The test color bar you see is a dummy placeholder when a needed codec is missing.

**Comment by RiantAnda** [comment]
@dariox86 
I can play videos from $HOME/.steam/steam/steamapps/common/Unravel/D3D11/V/ using totem(with x64 plugins 'base', 'good', 'bad'). Not sure which gstreamer plugins are actually needed and if the game uses x86 or x64 gstreamer. 
I can't install `lib32-gst-plugins-bad` because it is not in the regular arch linux repositories. This could be a reason for my issue.

**Comment by dariox86** [comment]
@RiantAnda, I only use regular repositories. You should need no packages from AUR.

**Comment by ghost** [comment]
The problem persists on Proton 6.3-5 and current experimental. Origin install perfectly but cant launch the game

GPU: RTX 3080
Nvidia 470.42.01
Kernel version: 5.13-tkg-bmq
Proton version: 6.3-5

**Comment by ranplayer** [comment]
Game starts but get stuck on the EA intro video with **Proton 7.0**:

GPU: GTX 970
Driver: 510.47.03
Kernel: 5.15 (Futext 2 patched)
O.S: Manjaro

[steam-1225560-proton7.tar.gz](https://github.com/ValveSoftware/Proton/files/8087634/steam-1225560-proton7.tar.gz)

**Comment by thePaulRichard** [comment]
Hello, I have no lucky playing this game. After the game start, crash before the main menu. I tried with multiple proton versions.

[steam-1225560.log](https://github.com/ValveSoftware/Proton/files/8836954/steam-1225560.log)

**Comment by hcwinsemius** [comment]
Hi Paulo,

What sort of hardware do you have? That may help to figure out what the
problem is. I haven't placed for a while but got it working and it worked
perfectly. I'll give it a try on my setup and get back to you.

Cheers

Op za 4 jun. 2022 05:42 schreef Paulo Paim ***@***.***>:

**Comment by thePaulRichard** [comment]
Hello @hcwinsemius !

I'm using a Laptop with hybrid graphics:
I'm trying some parameters, with `PROTON_USE_WINED3D=1 %command%` the game seems to start normally.


[hardware.txt](https://github.com/ValveSoftware/Proton/files/8837979/hardware.txt)

**Comment by hcwinsemius** [comment]
I notice you have an AMD dGPU. It may be that the game doesn't run automatically on the dGPU but revers to the on-board intel because it cannot find your amd vulkan drivers. Possibly you may have to superimpose those. Can you try running as follows:

```
DISABLE_LAYER_AMD_SWITCHABLE_GRAPHICS_1=1 VK_ICD_FILENAMES=/usr/share/vulkan/icd.d/radeon_icd.i686.json:/usr/share/vulkan/icd.d/radeon_icd.x86_64.json %command%
```

**Comment by thePaulRichard** [comment]
@hcwinsemius I tried running with the parameters but still the same problem, I also tried with `DRI_PRIME=1 %command%`

With `PROTON_USE_WINED3D=1 %command%` runs almost the time, sometimes the game crash before the main menu, if I tried to lauch again, works fine.

**Comment by hcwinsemius** [comment]
Ok, I think I got it. I tried several things but ended up with this:

- Install as usual. Things will not run.
- add quartz to your wine env with protontricks. If you have protontricks
installed it is a matter of opening up a terminal and typing

```
protontricks 1225560 quartz
```

- switch to proton experimental
- run without any special arguments

Et voila!

Does that work?



Op za 4 jun. 2022 17:47 schreef Paulo Paim ***@***.***>:

**Comment by hcwinsemius** [comment]
Ok, I think I got it. I tried several things but ended up with this:

Install as usual. Things will not run.

Add quartz to your wine env with protontricks. If you have protontricks installed it is a matter of opening up a terminal and typing 
```
protontricks 1225560 quartz
```
Switch to proton experimental Run without any special arguments. Et voila! 

Does that work?

**Comment by thePaulRichard** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/3961#issuecomment-1146761800

Hello @hcwinsemius ! 

I played three times after use protontricks to install quartz and it's working, I'm still testing, in the first time, after finished the first chapter, the game crashes, but after that won`t crashes anymore.

Thank you for now!!

By the way, this 'trick' (using protontricks), also works with Unravel Two.

**Comment by hcwinsemius** [comment]
Cool! I will post the solution on protondb for a wider use.

**Comment by ranplayer** [comment]
I was able to finish the game with Proton Experimental (latest version tested from **22/07/28**). The only annoying issue I had was for launching the game: every time it pops up a black window and then it self-minimizes to the system tray. After that you just have to click the minimized window and the game works normally. This may not be alarming for players using the Steam desktop interface, but for players using the BigPicture mode is a bad experience. I had always to switch to desktop mode and use the mouse to click on the minimized window. 

I can't confirm this is a Proton only issue, because I haven't played the game on Windows. It seems not to be DXVK's fault either, because I've tried to launch the game with WineD3D and it behaves the same.

Proton Experimental log: 
[steam-1225560.tar.gz](https://github.com/ValveSoftware/Proton/files/9226490/steam-1225560.tar.gz)


My specs:
- O.S: Arch
- Kernel: 5.18.12
- CPU: Ryzen 7 5700G
- GPU: GTX 970 4Gb (Driver: 515.57)
- RAM: 16 Gb

**Comment by Salaheddine240** [comment]
Steam tools إجباري 😂
